src/ImageRunModal.jsx

```diff
@@ -417,17 +423,20 @@ export class ImageRunModal extends React.Component {
                                      default={{ envKey: null, envValue: null }}
                                      itemcomponent={ <EnvVar />} />
                 </FormGroup>
+                <FormGroup fieldId='run-image-dialog-run' label={_("Start after creation")}>
```

Comment:
The fieldId has to be the same with the id of the field that is nested in the FormGroup. So in this case 'run'.
Please use more complicated ids however, not 'run' as in one page ids should be unique. and single word string tend to become not unique :)